client/agent.go

```diff
@@ -188,6 +188,7 @@ func (c *FuzzitClient) runLibFuzzerFuzzing() error {
 		"corpus",
 		"seed",
 	}
+	args = append(args, strings.Split(c.currentJob.Args, " ")...)
```

Comment:
@bayandin Thanks for the fix (probably missed out while we open-sourced/rewrote our agent). are you sure this will work and the additional libfuzzer arguments shouldn't be before the corpus and seed directory?

Reply:
Thanks, fixed!